packages/packagers/html/src/HTMLPackager.js

```diff
@@ -112,6 +114,51 @@ async function getAssetContent(
   return null;
 }
 
+async function addIntrinsicImageSizes(
+  bundle: NamedBundle,
+  bundleGraph: BundleGraph<NamedBundle>,
+  tree,
+) {
+  const imageNodes = [];
+  tree.walk(node => {
+    if (node.tag === 'img' && node.attrs && node.attrs['src']) {
+      imageNodes.push(node);
+    }
+    return node;
+  });
+
+  for (const node of imageNodes) {
+    const dependency = bundle
+      .getMainEntry()
+      ?.getDependencies()
+      .find(bundle => bundle.id === node.attrs['src']);
+
+    if (!dependency) {
+      continue;
+    }
+
+    const image = bundleGraph.getResolvedAsset(dependency);
+
+    if (!image) {
+      continue;
+    }
+
+    if (
+      typeof image.meta.width !== 'number' &&
+      typeof image.meta.height !== 'number'
```

Comment:
Are these supposed to be set somewhere (e.g. the image transformer)?

Reply:
Ideally, yes.